keepalived/check/check_api.c

```diff
@@ -117,6 +120,30 @@ queue_checker(void (*free_func) (void *), void (*dump_func) (void *)
 	}
 }
 
+int
```

Comment:
I think this should be bool, return false if compare fails (currently returning -1), and true if compares the same.